__tests__/articles.test.js

```diff
@@ -90,3 +90,114 @@ describe("GET/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET/api/articles/:article_id/comments", () => {
+  test("responds with a status of 200, returns an array of comments with the corrosponding article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("responds with a status of 200, returns an array of comments with the corrosponding article_id sorted by the newest comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
```

Comment:
`expect(comments).toBeSortedBy("created_at", { descending: true });` is the only assertion you need here as the rest is already covered in the test above - there's no need to repeat code